### PR TITLE
Prl 5323 c7 document welsh

### PR DESCRIPTION
**Pull request by khurshedk**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRL-5323

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/citizen/CaseApplicationResponseController.java [hunk 1]
```diff
@@ -48,13 +48,14 @@ public Document generateC7DraftDocument(
         @PathVariable("caseId") String caseId,
         @PathVariable("partyId") String partyId,
         @RequestHeader(HttpHeaders.AUTHORIZATION) @Parameter(hidden = true) String authorisation,
+        @RequestHeader("isWelsh") boolean isWelsh,
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/controllers/citizen/CaseApplicationResponseController.java:51]: Why this as a parameter? We need to generate welsh only if there is a welsh check at the case level

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/citizen/CaseApplicationResponseController.java [hunk 2]
```diff
@@ -48,13 +48,14 @@ public Document generateC7DraftDocument(
...
         @RequestHeader("serviceAuthorization") String s2sToken) throws Exception {
 
         CaseDetails caseDetails = coreCaseDataApi.getCase(authorisation, s2sToken, caseId);
         CaseData caseData = CaseUtils.getCaseData(caseDetails, objectMapper);
 
         caseApplicationResponseService.updateCurrentRespondent(caseData, YesOrNo.Yes, partyId);
-        return caseApplicationResponseService.generateC7DraftDocument(authorisation, caseData);
+        return caseApplicationResponseService.generateC7DraftDocument(authorisation, caseData,isWelsh);
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/controllers/citizen/CaseApplicationResponseController.java:58]: format please. also dont think its needed

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/citizen/CaseApplicationResponseController.java (hunk 1 above)
Comment (AlokDatta) [reply src/main/java/uk/gov/hmcts/reform/prl/controllers/citizen/CaseApplicationResponseController.java:51]: Understood and agreed

File: src/main/java/uk/gov/hmcts/reform/prl/services/CaseApplicationResponseService.java [hunk 3]
```diff
@@ -214,7 +222,8 @@ private CaseData addCitizenDocumentsToTheQuarantineList(CaseData caseData, List<
     private String getCategoryId(Element<Document> element) {
 
         if (null != element.getValue().getDocumentFileName()) {
-            if (element.getValue().getDocumentFileName().equalsIgnoreCase("C7_Document.pdf")) {
+            if (element.getValue().getDocumentFileName().equalsIgnoreCase("C7_Document.pdf")
+                || element.getValue().getDocumentFileName().equalsIgnoreCase("Final_C7_response_Welsh.pdf")) {
```
Comment (AlokDatta) [inline src/main/java/uk/gov/hmcts/reform/prl/services/CaseApplicationResponseService.java:226]: Switch case

File: src/main/java/uk/gov/hmcts/reform/prl/controllers/citizen/CaseApplicationResponseController.java (hunk 2 above)
Comment (khurshedk) [reply src/main/java/uk/gov/hmcts/reform/prl/controllers/citizen/CaseApplicationResponseController.java]: formatted text  and  as discussed this flag is required to generate draft C7

File: src/main/java/uk/gov/hmcts/reform/prl/services/CaseApplicationResponseService.java (hunk 3 above)
Comment (khurshedk) [reply src/main/java/uk/gov/hmcts/reform/prl/services/CaseApplicationResponseService.java]: converted code to switch case.

**Review by AlokDatta — APPROVED** [review]

**Pull request merged**